### DISCUSSION TITLE
Memoria Ram Xamarin

### DISCUSSION THREAD

**Discussion by Josecer10** [community/community#49352] [Programming Help]
Hola Necesito Desarrollar un Programa pequeño, Se trata sobre una aplicación en Xamarin para Moviles/ Celulares.

Sucede que estoy empezando y conozco sobre python, intermedio. aunque en xamarin sinceramente es muy poco lo que puedo saber.
Y necesito desarrollar esta App Para la prueba de Semestre y tengo 5 dias para resolverla y entregarla por completa.

Me exigen que realice una App para Movil y que ella pueda acceder a la memoria ram y en listar lo que en ella hay y asi poder extraer 1 archivo. 

Amigos yo no se como hacer eso :/ que puedo hacer para entender y pasar mi prueba Les agradezco.

**Comment by rauzerfabiano** [comment]
Hola,

Para desarrollar una aplicación en Xamarin para acceder a la memoria RAM y listar los archivos, debes seguir los siguientes pasos:

    Crear un nuevo proyecto de Xamarin en Visual Studio.
    Agregar los permisos necesarios en el archivo AndroidManifest.xml para acceder a la memoria RAM y listar los archivos.
    Utilizar la clase File para acceder a los archivos del dispositivo y la clase MemoryInfo para obtener información sobre la memoria RAM.
    Utilizar una interfaz de usuario para mostrar la lista de archivos y permitir al usuario seleccionar el archivo que desea extraer.

Para aprender más sobre cómo desarrollar aplicaciones en Xamarin, te recomiendo revisar la documentación oficial de Xamarin y hacer tutoriales en línea para adquirir más experiencia en el desarrollo de aplicaciones móviles.

Espero que esto te ayude a entender cómo desarrollar tu aplicación y a pasar tu prueba con éxito. ¡Buena suerte!

**Reply by Josecer10** [reply]
Hola 

Como has estado fabiananorauzer-egsys Te agradezco tu gran ayuda en estos momentos, de hecho ya tengo todo practicamente listo, despues de pasar mi prueba que estoy seguro saldra con exito, gracias a tu ayuda tambien. Ya mas adelante le quiero agregar algunos detalles en mente que tengo y la publicare para las personas que quieran utilizarla. 

gracias. bendiciones.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Josecer10 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
